<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StaffTextPropertiesDialog</class>
 <widget class="QDialog" name="StaffTextPropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabSwingSettings">
      <attribute name="title">
       <string>Swing Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="setSwingBox">
         <property name="title">
          <string>Swing Settings</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <widget class="QWidget" name="horizontalLayoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>601</width>
            <height>31</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QLabel" name="SwingLabel">
             <property name="text">
              <string>Swing:</string>
             </property>
             <property name="buddy">
              <cstring>swingOff</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="swingOff">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Off</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="swingEighth">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Eighth note</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="swingSixteenth">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Sixteenth note</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="horizontalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>601</width>
            <height>41</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_16">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Select swing ratio:</string>
             </property>
             <property name="buddy">
              <cstring>swingBox</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="swingBox">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="suffix">
              <string>%</string>
             </property>
             <property name="minimum">
              <number>25</number>
             </property>
             <property name="value">
              <number>60</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>setSwingBox</tabstop>
  <tabstop>swingOff</tabstop>
  <tabstop>swingEighth</tabstop>
  <tabstop>swingSixteenth</tabstop>
  <tabstop>swingBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>StaffTextPropertiesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>578</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StaffTextPropertiesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>578</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
